.github/pull_request_template.md

```diff
@@ -28,6 +28,6 @@ Applicable spec: <link>
 - [ ] The documentation is generated using `src-docs`
 - [ ] The documentation for charmhub is updated
 - [ ] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)
-- [ ] The [changelog](../CHANGELOG.md) has been updated
+- [ ] The docs/changelog.md is updated with user-relevant changes in the format of [keep a changelog v1.1.0](https://keepachangelog.com/en/1.1.0/)
```

Comment:
Could this please be changed to [changelog](docs/changelog.md) so that the link is clickable

Reply:
Updated to `[changelog](../docs/changelog.md)`.
Tested in vscode and the link works.